.github/workflows/main.yml

```diff
@@ -1,17 +1,32 @@
-name: CI
+name: meetup-talks
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v1
-    - name: Run a one-line script
-      run: echo Hello, world!
-    - name: Run a multi-line script
-      run: |
-        echo Add other actions to build,
-        echo test, and deploy your project.
+      - name: Setup linkstatus, mdspell and write-good
+        run: |
+          python3 -m pip install linkstatus --user
+          sudo apt install nodejs
+          sudo npm i markdown-spellcheck -g
+          sudo npm install write-good -g
+
+      - name: Checkout to changes
+        uses: actions/checkout@v1
+
+      - name: Check Links
+        run: |
+          changes=`git diff remotes/origin/${{ github.base_ref }} --name-only`
+          ~/.local/bin/linkstatus --recursive --timeout 10 --retry 5 $changes
+
+      - name: Spelling Suggestions
+        run: |
+          changes=`git diff remotes/origin/${{ github.base_ref }} --name-only`
```

Comment:
Please use `$()` at all the places to run the command instead of using backticks

Reply:
hummm. I always fallow `backticks` for single line command but good to have `$()` for multi-line command

Reply:
This should give more information: https://google.github.io/styleguide/shell.xml?showone=Command_Substitution#Command_Substitution
https://stackoverflow.com/q/9449778/6202405